src/components/plot/bookmarkbutton.scss

```diff
@@ -1,9 +1,16 @@
+@import './plot.scss';
```

Comment:
@kanitw Do you think it's better to import it or should I copy and paste the command class from plot.scss?